src/gfal_opers.c

```diff
@@ -39,12 +40,12 @@ size_t s_local_mount_point=0;
 gboolean guid_mode=FALSE;
 
 void gfalfs_set_local_mount_point(const char* local_mp){
-	g_strlcpy(local_mount_point, local_mp, 2048);
+    g_strlcpy(local_mount_point, local_mp, 2048);
 	s_local_mount_point= strlen(local_mount_point);
 }
 
 void gfalfs_set_remote_mount_point(const char* remote_mp){
-	g_strlcpy(mount_point, remote_mp, 2048);
+    g_strlcpy(mount_point, remote_mp, 2048);
 	s_mount_point= strlen(remote_mp);
 }
```

Comment:
I was thinking about that, It didn't appear in my local directory after my commit !